src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -111,4 +111,25 @@ class Wallet(
   fun removeAccountSigner(sourceAddress: String, signerAddress: String): Transaction {
     return addAccountSigner(sourceAddress, signerAddress, 0)
   }
+
+  // Submit transaction
+  fun submitTransaction(signedTransaction: Transaction, serverInstance: Server = server): Boolean {
```

Comment:
can we change the name of `serverInstance` to `server`, and change the object's `server` object to `defaultServer`?

Reply:
Not really... The `server` is defined for the whole class and it doesn't make sense to name it `defaultServer` in other places where it is used. This `serverInstance` is only used for tests and I couldn't come up with a better name. :thinking: 

![image](https://user-images.githubusercontent.com/7346473/190259803-b924a9a8-55e1-4a30-aab9-1dd673aa3642.png)

Reply:
Oh hmm.. I would replace the `horizonUrl` parameter with a `server` parameter then, what way you provide a way to pass custom servers in tests but don't have a custom parameter on one of the wallet functions.

Reply:
`Wallet` class might change but I think it's too early to optimize it now. We don't have more significant pieces yet (create and recover accounts) that might drive some of the decisions.
For an average user, it might be easier (at least to me) to pass a `horizonUrl` rather than `server`. We can discuss this later with more people. :slightly_smiling_face: